### PR TITLE
[WFLY-19704] Update read-operation-names and read-operation-descripti…

### PR DESCRIPTION
**Pull request by bstansberry**
…on example

https://issues.redhat.com/browse/WFLY-19704

### CODE REVIEW

**Comment by bstansberry** [comment]
This is docs only so I'm not going to bother re-running the jobs that failed with various intermittent problems.

File: docs/src/main/asciidoc/_admin-guide/Management_resources.adoc [hunk 1]
```diff
@@ -123,17 +147,35 @@ details of one of them, via the CLI one would:
...
+            "max-length" => 2147483647L,
+            "web-url" => true,
+            "stability" => "default"
         }},
         "reply-properties" => {
             "type" => BYTES,
             "description" => "The hash of managed deployment content that has been uploaded to the domain's or standalone server's deployment content repository.",
-            "min-length" => 20,
-            "max-length" => 20,
-            "nillable" => false
```
Comment (soul2zimate) [inline docs/src/main/asciidoc/_admin-guide/Management_resources.adoc:135]: Is this intentional deletion? 

`"nillable" => false,` is included in the CLI output.

Comment (bstansberry) [reply docs/src/main/asciidoc/_admin-guide/Management_resources.adoc:135]: @soul2zimate You're right.

File: docs/src/main/asciidoc/_admin-guide/Management_resources.adoc [hunk 2]
```diff
@@ -123,17 +147,35 @@ details of one of them, via the CLI one would:
...
+            "nillable" => true,
+            "alternatives" => [
+                "input-stream-index",
+                "hash",
+                "bytes",
+                "path",
+                "relative-to",
+                "empty"
+            ],
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_admin-guide/Management_resources.adoc:160]: @soul2zimate I realize now when I filed https://issues.redhat.com/browse/WFCORE-6981 I was overly focused on the 'reply-properties' output. This nillable=true and 'alternatives' stuff is wrong as well, for the same basic reason -- the OperationDescription is reusing an AttributeDefinition that's meant for a different op.

File: docs/src/main/asciidoc/_admin-guide/Management_resources.adoc (hunk 1 above)
Comment (bstansberry) [reply docs/src/main/asciidoc/_admin-guide/Management_resources.adoc:135]: Fixed.

File: docs/src/main/asciidoc/_admin-guide/Management_resources.adoc (hunk 2 above)
Comment (bstansberry) [reply docs/src/main/asciidoc/_admin-guide/Management_resources.adoc]: @soul2zimate I updated this bit to use 'nillable => false' and no alternatives, as that's the correct data and will be what's reported when https://issues.redhat.com/browse/WFCORE-7066 is fixed.

File: docs/src/main/asciidoc/_admin-guide/Management_resources.adoc (hunk 1 above)
Comment (bstansberry) [reply docs/src/main/asciidoc/_admin-guide/Management_resources.adoc:135]: Thanks for noticing this; that's what got me thinking about how the request-properties weren't correct.

**Review by soul2zimate — APPROVED** [review]
Hi @bstansberry  Yes, the output is looks good to me.

**Comment by bstansberry** [comment]
Thanks @soul2zimate

**Pull request merged**